packages/functions/src/common/BookAppointmentHelper.ts

```diff
@@ -15,11 +15,19 @@ import {
 import { notifyOnAppointmentBooked } from "../notifications/BookAppointmentNotifier";
 import { DbDonor, DbAppointment } from "../function-types";
 import * as DbAppointmentUtils from "../utils/DbAppointmentUtils";
+import { BookAppointmentStatus } from "common/src/functions-api";
 
-export interface ValidBookAppointmentResponse {
+export type ValidBookAppointmentResponse =
+  | ValidBookAppointment
+  | InvalidBookAppointment;
+
+export type ValidBookAppointment = {
```

Comment:
I don't think you need to export this one (and the next one)

Reply:
Done

Reply:
I see you changed only one

Reply:
The ValidBookAppointment is needed as an import for the test file